ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -542,7 +542,6 @@ library unstable-cardano-tools
     , transformers-except
 
 executable db-analyser
-  import:         common-exe
```

Comment:
We probably still want to import at least `common-lib` to get eg various warning flags:
https://github.com/IntersectMBO/ouroboros-consensus/blob/49181c9fea384b4fbb144f1ec86dabdf72b99e84/ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal#L35-L44